packages/cmf/src/cmfConnect.js

```diff
@@ -111,7 +111,7 @@ export function getStateToProps({
 
 	let userProps = {};
 	if (mapStateToProps) {
-		userProps = mapStateToProps(state, ownProps, cmfProps);
+		userProps = mapStateToProps(state, { ...viewProps, ...ownProps }, cmfProps);
```

Comment:
see with @jmfrancois, overrides the props by cmf settings. We have to override the props by the settings. Just add a doc on this.